src/devices/src/display/gtk/worker.rs

```diff
@@ -0,0 +1,278 @@
+use std::os::fd::AsRawFd;
+
+use crate::display::gtk::DisplayEvent;
+use crate::display::{DisplayInfo, DisplayInfoList, MAX_DISPLAYS};
+use utils::pollable_channel::PollableChannelReciever;
+
+use glib::{source, ControlFlow, IOCondition};
+use gtk4::{
+    gdk,
+    gio::{ActionEntry, Cancellable, SimpleActionGroup},
+    prelude::{ActionMapExtManual, GtkWindowExt, WidgetExt},
```

Comment:
```suggestion
    prelude::*,
```
as stuff can move from ExtManual to Ext with time, so avoid potential code fixing with version bumps